### PR TITLE
kb(grid): Update CheckBox KB after customer feedback

### PR DESCRIPTION
**Pull request by dimodi**

### CODE REVIEW

File: knowledge-base/grid-checkbox-editing.md [hunk 1]
```diff
@@ -26,17 +26,14 @@ res_type: kb
...
-How to update boolean values in the Grid with no edit mode?
-
-How to toggle checkboxes in the Grid directly without using the built-in editing feature?
-
-The Grid is not editable, but it should allow users to check and uncheck check box values.
-
-How to use a single click on Grid checkboxes to set the opposite value?
-
-How to get a checkbox value in the Grid for update action on button click?
+* To edit a checkbox in a Grid cell, users need to click once to put the row (or cell) in edit mode. They they need another click to change the checkbox value. How to do this with only one click?
```
Comment (ikoevska) [inline knowledge-base/grid-checkbox-editing.md:31]:
```suggestion
* To edit a checkbox in a Grid cell, users need to click once to put the row (or cell) in edit mode. Then they need another click to change the checkbox value. How to do this with only one click?
```

File: knowledge-base/grid-checkbox-editing.md [hunk 2]
```diff
@@ -26,17 +26,14 @@ res_type: kb
...
+* How to update boolean values in the Grid with no edit mode?
+* How to toggle checkboxes in the Grid directly without using the built-in editing feature?
+* The Grid is not editable, but it should allow users to check and uncheck check box values.
+* How to use a single click on Grid checkboxes to set the opposite value?
+* How to get a checkbox value in the Grid for update action on button click?
```
Comment (ikoevska) [inline knowledge-base/grid-checkbox-editing.md:36]: Just a suggestion based on our conversation from Teams. If it doesn't make sense, discard it.

```suggestion
* How to get the value of a checkbox and use it for an update action on button click?
```

**Review by ikoevska — APPROVED** [review]

File: knowledge-base/grid-checkbox-editing.md [hunk 3]
```diff
@@ -26,17 +26,14 @@ res_type: kb
...
-
-The Grid is not editable, but it should allow users to check and uncheck check box values.
-
-How to use a single click on Grid checkboxes to set the opposite value?
-
-How to get a checkbox value in the Grid for update action on button click?
+* To edit a checkbox in a Grid cell, users need to click once to put the row (or cell) in edit mode. Then they need another click to change the checkbox value. How to do this with only one click?
+* How to update boolean values in the Grid with no edit mode?
+* How to toggle checkboxes in the Grid directly without using the built-in editing feature?
+* The Grid is not editable, but it should allow users to check and uncheck check box values.
```
Comment (dimodi) [inline knowledge-base/grid-checkbox-editing.md:34]:
```suggestion
* How to allow users to check and uncheck check box values when the Grid is not editable?
```

File: knowledge-base/grid-checkbox-editing.md [hunk 4]
```diff
@@ -26,17 +26,14 @@ res_type: kb
...
-The Grid is not editable, but it should allow users to check and uncheck check box values.
-
-How to use a single click on Grid checkboxes to set the opposite value?
-
-How to get a checkbox value in the Grid for update action on button click?
+* To edit a checkbox in a Grid cell, users need to click once to put the row (or cell) in edit mode. Then they need another click to change the checkbox value. How to do this with only one click?
+* How to update boolean values in the Grid with no edit mode?
+* How to toggle checkboxes in the Grid directly without using the built-in editing feature?
+* How to allow users to check and uncheck checкbox values when the Grid is not editable?
+* How to use a single click on Grid checkboxes to set the opposite value?
```
Comment (dimodi) [inline knowledge-base/grid-checkbox-editing.md:35]:
```suggestion
* How to use a single click on Grid checkboxes to set the reverse (opposite) value?
```

**Pull request merged**